### PR TITLE
Add more of the WebSocket API

### PR DESCRIPTION
**Pull request by garyb**

### CODE REVIEW

File: src/DOM/Websocket/BinaryType.purs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+instance boundedBinaryType :: Bounded BinaryType where
+  bottom = Blob
+  top = ArrayBuffer
+
+instance enumBinaryType :: Enum BinaryType where
+  succ = defaultSucc toEnumBinaryType fromEnumBinaryType
+  pred = defaultPred toEnumBinaryType fromEnumBinaryType
+
+instance boundedEnumBinaryType :: BoundedEnum BinaryType where
+  cardinality = Cardinality 3
```
Comment (MaloJaffre) [inline src/DOM/Websocket/BinaryType.purs]: This cardinality is incorrect (1).

File: src/DOM/Websocket/ReadyState.purs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+instance boundedReadyState :: Bounded ReadyState where
+  bottom = Connecting
+  top = Closed
+
+instance enumReadyState :: Enum ReadyState where
+  succ = defaultSucc toEnumReadyState fromEnumReadyState
+  pred = defaultPred toEnumReadyState fromEnumReadyState
+
+instance boundedEnumReadyState :: BoundedEnum ReadyState where
+  cardinality = Cardinality 3
```
Comment (MaloJaffre) [inline src/DOM/Websocket/ReadyState.purs]: This cardinality is incorrect (2).

**Review by MaloJaffre — COMMENTED** [review]
Two incorrect cardinalities

**Comment by garyb** [comment]
Argh! Thank you.

**Pull request merged**